images/win/toolsets/toolset-2022.json

```diff
@@ -304,5 +304,9 @@
     },
     "node": {
         "default": "14.18.1"
+    },
+    "Mysql": {
+        "short_version": "8",
+        "full_version": "8.0.26"
```

Comment:
Is it possible to download the latest 8.x.x version instead of hardcoding the exact one?

Reply:
Yes, I made changes for this